src/main/java/net/minecraftforge/client/event/MovementUpdateEvent.java

```diff
@@ -27,11 +27,11 @@
  * This event is fired after player movement inputs are updated.<br>
  * Handlers can freely manipulate {@link MovementInput} to cancel movement.<br>
  */
-public class InputUpdateEvent extends PlayerEvent
+public class MovementUpdateEvent extends PlayerEvent
```

Comment:
I think `MovementInputUpdateEvent` would be better, since `MovementUpdateEvent` could mean all kinds of things.

Reply:
That does sound better, and more reflects the purpose of the event. Will do.